task-requests/details/script.js

```diff
@@ -564,7 +564,14 @@ function populateModalContent(index) {
     'data-modal-description-value',
     'proposed-description-value',
   );
-  descriptionValue.textContent = userData.description;
+
+  if (userData?.markdownEnabled ?? false) {
```

Comment:
The original code used innerHTML to set the rendered Markdown, which could be vulnerable to Cross-Site Scripting (XSS) attacks if userData.description contains untrusted content.

Reply:
Done!